draft-ietf-tls-tls13.md

```diff
@@ -1648,16 +1633,14 @@ Clients MUST NOT use cookies in subsequent connections.
 ###  Signature Algorithms
 
 The client uses the "signature_algorithms" extension to indicate to the server
-which signature algorithms may be used in digital signatures.
-
-Clients which offer one or more cipher suites which use certificate authentication
-(i.e., any non-PSK cipher suite) MUST send the "signature_algorithms" extension.
```

Comment:
That is, the condition for when the client MUST send signature_algorithms is different, but if the client wants the server to authenticate with a cert, they had better send it.